utilities/additionalProperties.go

```diff
@@ -1,34 +1,68 @@
 package utilities
 
-import "encoding/json"
+import (
+	"encoding/json"
+	"errors"
+	"strings"
+)
+
+func ValidateAdditionalProperty[T any](dstMap map[string]T, keysToRemove ...string) error {
+	containsKey := func(key string) bool {
+		for _, tag := range keysToRemove {
+			if tag == key {
+				return true
+			}
+		}
+		return false
+	}
+
+	for key, _ := range dstMap {
+		if strings.TrimSpace(key) == "" {
+			return errors.New("an additional property key can not be empty or whitespace")
```

Comment:
Haven't we decided on allowing whitespaces and empty strings in additionalProperty keys?

Reply:
Yes we decided to allow empty and white spaces. I will update it.